aries-site/src/pages/foundation/color.js

```diff
@@ -78,8 +104,68 @@ const Color = () => (
           </Box>
         )}
       </Subsection>
+      <Subsection name="Text Colors">
+        <Text>
+          Text needs to be legible in all contexts. Ensuring high contrast is
+          met for accessibility and readability is important to delivering users
+          a consistent reading experience throughout HPE.
+        </Text>
+        <AnchorCallToAction label="Typography Elements" href="#" />
```

Comment:
```suggestion
        <AnchorCallToAction label="Typography Elements" href="#" />
        </Subsection>
```
I think the Subsection should close here and the next Subsection follows it. The spacing for between Subsections is set up for them to be siblings as opposed to children/parents of each other.

Reply:
Mmmm... interesting. I think when I was writing this, I was imagining the parent `<Subsection />` to be a level 2 section (i.e. H2), then the inner `<Subsection />` being a level 3 section since that content is supporting and directly related to its parent. In this particular execution, the level 3 "Text Color Palette" would be accompanied by a sibling level 3 "Call To Action Text".

To me the supporting content feels like it should be children of the content it is supporting. Happy to adjust, but curious on your thoughts here.

Reply:
That makes sense as well, we would just need to think about how to make sure spacing is always consistent. The reason I thought they could just be siblings is because in html you could have h2, h3, etc at the same level as each other without needing to nest it inside of another div. I am open to allowing subsections to nest inside each other, but we would need to make sure spacing remains consistent.
```
<div>
<h2>Hello World</h2>
<p>This is my parapgraphy.</p>
<h3>Somthing else</h3>
</div>
...
```